src/key/mod.rs

```diff
@@ -307,15 +307,21 @@ impl PublicKey {
     ///
     /// Returns an error if the resulting key would be invalid.
     #[inline]
-    pub fn mul_tweak<C: Verification>(
-        mut self,
-        secp: &Secp256k1<C>,
-        other: &Scalar,
-    ) -> Result<PublicKey, Error> {
+    pub fn mul_tweak(mut self, other: &Scalar) -> Result<PublicKey, Error> {
+        // We have no seed here but we want rerandomiziation to happen for `rand` users.
```

Comment:
In f8bf19d9135d3c9cb42e6f6f751d6ead62ebc1e0:

There is no secret data. We don't want to rerandomize here. It slows things down for no purpose.

Reply:
This commit is still present in the latest push (but I can address it in a followup).

Reply:
My bad, I missed this comment yesterday. Fixed in #855